WooCommerce/NotificationExtension/OrderNotificationViewModel.swift

```diff
@@ -0,0 +1,154 @@
+import Foundation
+import KeychainAccess
+import Networking
+
+/// View Model for the `OrderNotificationViewController`type.
+///
+final class OrderNotificationViewModel {
```

Comment:
non-blocking: would be nice to have some test cases!

Reply:
Yeah, I wanted those but ran out of time in the HACK week 😇 
I have added an issue to the eagle backlog https://github.com/woocommerce/woocommerce-ios/issues/10683